content/styles/03-components/hero.css

```diff
@@ -10,6 +10,7 @@
   & h2,
   & h3 {
     color: var(--white);
+    text-shadow: 0 0 .3rem #555;
```

Comment:
We have a var for gray `--gray: #333;` do you think you could use it to replace `#555` ? 'cause they are pretty close. If not no problem ^^ or maybe change `--gray: #333;` for `--gray: #555;`

Reply:
other possible case we have an `--shadow-color: rgba(50, 50, 50, .15);` we could consider a `--text-shadow: 0 0 .3rem #555;` then `text-shadow: var(text-shadow: 0 0 .3rem #555);` In this case that could be re use (by me for exemple ^^)

Reply:
totally true!